data/json/recipes/recipe_deconstruction.json

```diff
@@ -2092,11 +2092,12 @@
     "components": [
       [ [ "processor", 1 ] ],
       [ [ "RAM", 4 ] ],
+      [ [ "circuit", 1 ] ],
       [ [ "cable", 6 ] ],
       [ [ "amplifier", 1 ] ],
+      [ [ "power_supply", 1 ] ],
       [ [ "plastic_chunk", 6 ] ],
-      [ [ "small_lcd_screen", 1 ] ],
-      [ [ "small_storage_battery", 2 ] ]
+      [ [ "large_lcd_screen", 1 ] ]
```

Comment:
The laptop is 2.7kg and has a volume of 2.5L, and the "large_lcd_screen" is an item that weighs 13kg, and has a volume of 15L, so it doesn't make sense to get that from disassembling it. I would revert back to a small_lcd_screen (0.5kg, 0.25L).